starfish/core/spacetx_format/test_coordinates.py

```diff
@@ -4,8 +4,9 @@
 from .util import SpaceTxValidator
 
 package_name = "starfish"
+current_version = "0.1.0"
```

Comment:
https://github.com/spacetx/slicedimage/pull/95

Reply:
You see keeping the coordinates.json schema in sync with the slicedimage version?

Reply:
That's the plane.

Reply:
:+1: Suggestion in f1c6078f7ec4f6f234b6bd52e3788389ecf49e9c